### DISCUSSION TITLE
Lerna monorepo Vercel deploy

### DISCUSSION THREAD

**Discussion by liqueflies** [nuxt/nuxt#17727] [Questions]
Hello folks,

I've a monorepo of a project like:

```
packages/
  - @repo/ui
  - @repo/website
```

My scripts are:

```
build: "lerna run build"
start: "lerna run start --scope=@repo/website"
```

I've tried to deploy on vercel, the build is fine but when look for output directory it fails.
I guess that configuration for vercel is auto-detecting so I am not able to understand what I am missing.

Any suggestions?

**Comment by Killian-Jeusselin** [comment]
Hi, I've recently encountered the same issue when trying to deploy a nuxt 3 monorepo with lerna + yarn workspaces. I was unable to deploy by adapting the "Build & Development Settings" section of Vercel. 
Same result you describe, build is fine, but the output directory is missing.

Following the [documentation](https://v3.nuxtjs.org/docs/deployment/vercel), choosing the nuxt preset available in the select dropdown for deployment should enable the correct settings. 

I've tried the following in Vercel settings (Project Settings -> general) : 

- set "packages/your_package_name" under the "Root Directory" settings (with the "include source files outside of the Root ..." option checked)
- set "Build & Development Settings" on the Nuxt framework preset
- Redeploy

The build is ok and the directory is found. Maybe you should try to see if that fixes your problem.
 
Unfortunately I've the following error now : 
`ERROR	Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'h3' imported from /var/task/index.mjs`

And Vercel is showing this status : 

<img width="540" alt="Capture d’écran 2022-03-28 à 17 43 11" src="https://user-images.githubusercontent.com/29757956/160436112-cb2d3a2a-f290-4e2c-be55-8bf518477ce1.png">

I don't know if this is related, I'll continue my research.

**Reply by exodusanto** [reply]
I think is related to #4217, there are some issues on monorepo setup with node_modules resolves
